### PR TITLE
Modern 2020 rebrand

### PR DESCRIPTION
**Pull request by wrussell1999**
Facelift to the website...

I think I've caught all the bugs!

### CODE REVIEW

**Comment by wrussell1999** [comment]
Gonna add a bio to the homepage

**Comment by wrussell1999** [comment]
Few more tweaks I'd like to make:
- Make the `Resources` page buttons match home page
- Add intro paragraph to the bottom of the home page (using Jekyll template)
- Add social links to bottom of home page

Feel free to have a play @danielfspencer or @jac0bwilson  (in a separate branch)

**Comment by wrussell1999** [comment]
The font only works if you have it installed. Need to configure to use URL like the Summer Event site

**Comment by wrussell1999** [comment]
Header box too large due to being combined with nav bar

File: README.md [hunk 1]
```diff
@@ -17,4 +17,4 @@ $ bundle exec jekyll serve
 
 ## Contributors
 
-This website is maintained primarily by the Hackathans for Schools Association Core Team.
+This website is maintained primarily by the Hackathans for Schools Core Team.
```
Comment (jac0bwilson) [inline README.md]:
```suggestion
This website is maintained primarily by the Hackathons for Schools Core Team.
```

File: docs/sponsorship.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ hide: true
 
 ## Who are we?
 
-At the **Hackathons for School Association** our mission is to improve the technology opportunities in schools, and to introduce more 15 to 18-year-olds to the world of hackathons and technology. We are a group of students who study STEM subjects at universities in the UK.
+At **Hackathons for School** our mission is to improve the technology opportunities in schools, and to introduce more 15 to 18-year-olds to the world of hackathons and technology. We are a group of students who study STEM subjects at universities in the UK.
```
Comment (jac0bwilson) [inline docs/sponsorship.md]:
```suggestion
At **Hackathons for Schools** our mission is to improve the technology opportunities in schools, and to introduce more 15 to 18-year-olds to the world of hackathons and technology. We are a group of students who study STEM subjects at universities in the UK.
```

**Review by jac0bwilson — COMMENTED** [review]
Just correcting your spellings as usual ;)

These seem to be old typos though so can't necessarily blame you...

**Comment by wrussell1999** [comment]
Alright, only things remaining are:

- Home page intro paragraph
- Home page social links

**Comment by wrussell1999** [comment]
Ideally, the home page would be a bit more Jeklly but it's not crucial

File: docs/constitution.md [hunk 3]
```diff
@@ -36,24 +36,24 @@ This meeting will be carried out over a voice call, and all members will be noti
...
 - Members must vote on whether the dissolution will go ahead, a majority must be reached.
 - Finances must be in order:
     - Members must all be reimbursed for agreed expenditure.
-    - Remaining money should be given to a group with aims in line with those of the **HFSA**.
+    - Remaining money should be given to a group with aims in line with those of the **H4S**.
 
 ---
 
-This constitution has been agreed to by all members of the **Hackathons for Schools Association** and agreed by all members of the Core Team on the 23rd February 2020.
+This constitution has been agreed to by all members of the **Hackathons for Schools** and agreed by all members of the Core Team on the 23rd February 2020.
```
Comment (jac0bwilson) [inline docs/constitution.md:56]: Do you think this date should be updated given the changes?

Idk if its necessary or not

File: docs/cpp.md [hunk 4]
```diff
@@ -212,16 +212,16 @@ Child Abuse Concern Report.
...
+- Any action taken to relay sensitive information to any external authority will first be cleared by the Child Protection Officer / Designated Safeguarding Person at **H4S**. 
 
 ## Implementation 
 
 This policy will be implemented as of the 24/02/2020. William Russell, our Child Protection Officer, will be ensuring throughout our events that this policy is abided by at all times.
 
-We are committed to reviewing our policy and good practice before every event held by **HFSA**.
+We are committed to reviewing our policy and good practice before every event held by **H4S**.
 
 This policy was last reviewed on 24/02/2020:
```
Comment (jac0bwilson) [inline docs/cpp.md:226]: Again, may want to update the date (or not)

**Review by jac0bwilson — APPROVED** [review]
All looks good to me! I don't know if the documents will want their dates updating due to the rebrand? Something to think about...

**Comment by wrussell1999** [comment]
Yeah that's a really good point. Would you be able to update them? 👌

**Pull request merged**